### DISCUSSION TITLE
URLs for fully-insured

### DISCUSSION THREAD

**Discussion by RLTx1391** [CMSgov/price-transparency-guide#494] [General]
Self-funded groups need to post a URL on their site with the link to the MRFs, even when they have a third party generate and host. I do not think fully-insured groups need to do the same and post a URL, but want to double check as we're hearing some chatter.

**Comment by Devyandu** [comment]
You are right, the regulation is for "[health plans and issuers of group or individual health insurance](https://www.cms.gov/healthplan-price-transparency/plans-and-issuers)" to make the MRFs publicly available.
A full-insured group is neither a health plan nor an issuer.

In the case of full-insured groups, it is their health plan who would need to post the URL.

PS : I do not work for CMS.
